### PR TITLE
874256-UG documentation added for .NET MAUI Theme support

### PR DESCRIPTION
**Pull request by AarthiArjunan**
### **Description**

- Added UG content for the .NET MAUI theme support

#### **Added topics**

- Applying themes

- How to
- Keys

### CODE REVIEW

File: MAUI/Themes/HowTo.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: How to customize theming in Syncfusion controls
+description: Learn here about how to switch between light and dark themes in Syncfusion Maui controls and more details.
```
Comment (Chrispineagunja) [inline MAUI/Themes/HowTo.md:4]: description: Learn how to switch between light and dark themes in Syncfusion Maui controls and more details.

File: MAUI/Themes/HowTo.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+platform: maui
+control: General
+documentation: ug
+---
+
+# Switching between light theme and dark theme
+
+To switch between light and dark themes in Syncfusion Maui controls, refer to the following example code.
```
Comment (Chrispineagunja) [inline MAUI/Themes/HowTo.md:12]: Refer to the following example code to switch between light and dark themes in Syncfusion Maui controls.

File: MAUI/Themes/HowTo.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            {
+                theme.SfVisualTheme = SfVisuals.MaterialDark;
+            }
+        }
+     }
+}
+
+{% endhighlight %}
+
+The complete Theme switch sample is available in `this` link.
```
Comment (Chrispineagunja) [inline MAUI/Themes/HowTo.md:37]: The complete theme switch sample is available in `this` link.

File: MAUI/Themes/Keys.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: About Keys for Syncfusion Controls | Syncfusion
+description: This section contains the details about the keys which are mapped to the UI elements of all the controls
```
Comment (Chrispineagunja) [inline MAUI/Themes/Keys.md:4]: description: This section contains the details about the keys mapped to the UI elements of all the controls.

File: MAUI/Themes/Themes.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+title: Applying themes for Syncfusion controls
+description: This section explains about theming in Syncfusion controls and how to override the key values for further customization
+platform: maui
+control: General
+documentation: ug
+---
+
+# Applying Themes in Maui Controls
+
+Syncfusion themes allow you apply colors across all the Syncfusion controls with a uniform approach and provide consistent look and feel to your applications. This section covers the following topics:
```
Comment (Chrispineagunja) [inline MAUI/Themes/Themes.md:12]: Syncfusion themes allow you to apply colors across all the Syncfusion controls with a uniform approach and provide a consistent look and feel to your applications. This section covers the following topics:

File: MAUI/Themes/Themes.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+</Application>
+
+{% endhighlight %}
+
+![OverridingControlSpecificColors](Images/DarkTheme_ControlSpecific.png)
+
+## Creating your own theme
+
+As an alternative approach to the above methods, you can also create your own theme. To create own theme, first, you need to merge the resource, whose key name should be "ControlName" + "Theme" based on the controls, e.g., SfChartTheme and SfTextInputLayoutTheme. You can find this key for each control in this `documentation` . After merging this resource, you need to merge the required color resources based on the UI elements that need to be customized. You can find the keys and the UI elements to which they are mapped to all the controls in this `documentation`.
```
Comment (Chrispineagunja) [inline MAUI/Themes/Themes.md:97]: You can also create your own theme as an alternative approach to the above methods. To create your own theme, first, you need to merge the resource, whose key name should be "ControlName" + "Theme" based on the controls. For example, the SfChartTheme and SfTextInputLayoutTheme. You can find this key for each control in this `documentation.` After merging this resource, you need to merge the required color resources based on the UI elements that need to be customized. You can find the keys and the UI elements to which they are mapped to all the controls in this `documentation.`

File: MAUI/Themes/Themes.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+Using this approach, you can create your own theme to all the controls or for specific controls you need.
```
Comment (Chrispineagunja) [inline MAUI/Themes/Themes.md:98]: Using this approach, you can create your own theme for all the controls or specific controls you need.

**Review by Chrispineagunja — COMMENTED** [review]
Content review completed.

File: MAUI/Themes/HowTo.md (hunk 2 above)
Comment (AarthiArjunan) [reply MAUI/Themes/HowTo.md:12]: Changes addressed.

File: MAUI/Themes/HowTo.md (hunk 3 above)
Comment (AarthiArjunan) [reply MAUI/Themes/HowTo.md:37]: Changes addressed.

File: MAUI/Themes/Keys.md (hunk 4 above)
Comment (AarthiArjunan) [reply MAUI/Themes/Keys.md:4]: Changes addressed.

File: MAUI/Themes/Themes.md (hunk 5 above)
Comment (AarthiArjunan) [reply MAUI/Themes/Themes.md:12]: Changes addressed.

File: MAUI/Themes/Themes.md (hunk 6 above)
Comment (AarthiArjunan) [reply MAUI/Themes/Themes.md:97]: Changes addressed.

File: MAUI/Themes/Themes.md (hunk 7 above)
Comment (AarthiArjunan) [reply MAUI/Themes/Themes.md:98]: Changes addressed.

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**